app/models/service.rb

```diff
@@ -0,0 +1,7 @@
+class Service < ApplicationRecord
+  has_many :service_users
+  has_many :user, through: :service_users
+  has_many :products
```

Comment:
Maybe I'm not understanding the nomenclature, but in our reality we have Services, which can be of different types (Gifts, Catering, etc...). What's this relationship between products and services? What do each of them represent?

Reply:
In my mind i was thinking that if a list of products like lunchboxes appear under the gift service, then it should be related (1 service to N products).

Did I interpreted the figma wrong?

Reply:
Can you tell me if it is now resolved?